app/javascript/components/pages/LastCallShow.js

```diff
@@ -1,9 +1,62 @@
-import React, { Component } from 'react'
+import React from 'react'
+import { Card, CardBody, CardTitle, CardImg,CardText, CardSubtitle, Button } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
+import { useParams } from 'react-router-dom'
 
-export default class LastCallShow extends Component {
-    render() {
-        return (
-            <div>LastCallShow</div>
-        )
-    }
+export default function LastCallShow(props) {
+  let { logged_in } = props
+  let { id } = useParams()
+  
+  let { cocktails } = props
+  let cocktail = cocktails.find(cocktailObj => cocktailObj.id == id)
+  
+
+  return (
+    <>                   
+    {cocktail && 
+      <Card>
+          <CardImg top style={{ width: '10rem' }} src={cocktail.image} alt="Card image cap" />
+          <CardBody>
+              <CardTitle>{cocktail.name}</CardTitle>
+              <CardSubtitle>Ingredients</CardSubtitle>
+              <CardText>
+                
+                  {cocktail.ingredients.map(ingredient => {
+                    return(
+                       <li> {ingredient} </li>
+                    )
+                  })
+                  }
+                
+              </CardText>
+              <CardSubtitle>Directions</CardSubtitle>
+              <CardText>
+                
+                  {cocktail.directions.map(direction => {
+                    return(
+                       <li> {direction} </li>
+                    )
+                  })
+                  }
+                
+              </CardText>
+              <NavLink to={`/`}>
+                  <Button> Home </Button>
+              </NavLink>
+              {logged_in &&
+              <>
+                <NavLink to={`/lastcalledit/${id}`}>
+                    <Button id="edit"> Edit </Button>
+                </NavLink>
+                
+                    <Button id="delete"> Delete </Button>
```

Comment:
Where will you be redirecting the page after the delete action? I see that the edit button has a link to the edit page. I do not see a link for your delete button.